app/src/App/index.js

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 
+import { ChakraProvider } from "@chakra-ui/react";
```

Comment:
are we still using chakra provider?

Reply:
No.  I removed it on the higher level file, but missed this one.  Thanks!